redwoods-insurance-mobile/redwoods-insurance/RestClient+NewClaim.swift

```diff
@@ -14,6 +14,7 @@ import SalesforceSDKCore
 // Extends SalesforceSDKCore.RestClient with convenience methods for working
 // with `Case` records.
 extension RestClient {
+	static let APIVERSION = "v46.0" // what api version to use for rest calls
```

Comment:
This should also be `47.0` for Winter '20.

Reply:
Switched to "v47.0"